src/Wrapper/Product.php

```diff
@@ -362,11 +362,13 @@ protected function addManufacturer(): void
         if (!Utils::isEmpty($manufacturerId)) {
             $manufacturer = $this->registryService->getManufacturer($manufacturerId);
 
-            if (Utils::isEmpty($manufacturer->getName())) {
+            if (Utils::isEmpty($manufacturer->getExternalName()) && Utils::isEmpty($manufacturer->getName())) {
                 return;
             }
 
-            $vendorAttribute = new Attribute('vendor', [$manufacturer->getName()]);
+            $manufacturerName = ($manufacturer->getExternalName()) ?: $manufacturer->getName();
```

Comment:
I prefer `??` over `?:`. We also may get rid of the braces. I like to add braces for boolean expressions, but in this case I don't really see the benefit :D

```suggestion
            $manufacturerName = $manufacturer->getExternalName() ?? $manufacturer->getName();
```

Reply:
?: checks for an empty strings but ?? - not. We are getting an empty string if the external name is not provided for manufacturer:
![image](https://user-images.githubusercontent.com/92298567/155346446-7c94bf60-483b-4b61-99d8-ec37a3039eda.png)

With ?? we are getting the following error:
Something went wrong. Message:                                         
         FINDOLOGIC\Export\Exceptions\EmptyValueNotAllowedException: Empty      
         values are not allowed for "attribute" values

Reply:
Wow.. typical Plenty :face_exhaling: